init.rb

```diff
@@ -16,8 +16,13 @@
   name 'Redmine Issue Badge plugin'
   author 'Akiko Takano'
   description 'Plugin to show the number of assigned issues with badge on top menu.'
-  version '0.0.2'
+  version '0.0.3'
   url 'https://github.com/akiko-pusu/redmine_issue_badge'
   author_url 'http://twitter.com/akiko_pusu'
   requires_redmine :version_or_higher => '2.5.0'
+
+  settings :partial => 'settings/redmine_issue_badge',
```

Comment:
__[rubocop]__
* Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)